packs/feat-effects/effect-crash-against-me-clang.json

```diff
@@ -0,0 +1,57 @@
+{
+    "_id": "sEWwGSuXxcTl020D",
+    "img": "icons/skills/melee/sword-damaged-broken-orange.webp",
+    "name": "Effect: Crash Against Me (Clang)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.actionspf2e.Item.Crash Against Me]</p>\n<p>You are off-guard and take a –2 circumstance penalty to attacks with the clanging weapon.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder War of Immortals"
+        },
+        "rules": [
+            {
+                "inMemoryOnly": true,
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.conditionitems.Item.Off-Guard"
+            },
+            {
+                "key": "RollOption",
```

Comment:
It might be possible use a ChoiceSet like this example from "Effect: Bespell Strikes":
```
{
	"choices": {
		"attacks": true
	},
	"flag": "weapon",
	"key": "ChoiceSet",
	"prompt": "PF2E.SpecificRule.Prompt.Weapon"
}
```

Reply:
Good idea. As an aside, this uses a slug as the selection, and it looks like Bespell Strikes is expecting an id, so the effect is probably not working. Worth taking a look at.